chart/values.yaml

```
@@ -129,20 +128,18 @@ commonConfiguration: |
   REACT_APP_CLIENT_ID="${CARTO_AUTH0_CLIENT_ID}"
   REACT_APP_AUTH0_DOMAIN="${CARTO_AUTH0_CUSTOM_DOMAIN}"
   REACT_APP_ACCOUNTS_API_URL="https://${ACC_DOMAIN}"
-  REACT_APP_ACCOUNTS_URL="https://${ONPREM_DOMAIN}/acc/"
-  REACT_APP_WORKSPACE_API_URL="https://${ONPREM_DOMAIN}/workspace-api"
-  REACT_APP_API_BASE_URL="https://${ONPREM_DOMAIN}/api"
-  REACT_APP_PUBLIC_MAP_URL="https://${ONPREM_DOMAIN}/workspace-api/maps/public"
+  REACT_APP_ACCOUNTS_URL="https://${SELFHOSTED_DOMAIN}/acc/"
+  REACT_APP_WORKSPACE_API_URL="https://${SELFHOSTED_DOMAIN}/workspace-api"
+  REACT_APP_API_BASE_URL="https://${SELFHOSTED_DOMAIN}/api"
+  REACT_APP_PUBLIC_MAP_URL="https://${SELFHOSTED_DOMAIN}/workspace-api/maps/public"
```

Can you also add the fixed url? It should be:
```suggestion
  REACT_APP_PUBLIC_MAP_URL="https://${SELFHOSTED_DOMAIN}/api/v3/maps/public"
```